packages/permissionless/actions/bundler/getUserOperationReceipt.ts

```diff
@@ -20,11 +20,14 @@ export type GetUserOperationReceiptParameters = {
 
 export type GetUserOperationReceiptReturnType = {
     userOpHash: Hash
+    entryPoint: Address
     sender: Address
     nonce: bigint
+    paymaster?: Address | undefined
```

Comment:
I don't think we need undefined if we have used `?:`

Reply:
good point, they have been removed. Although saw this pattern on some other types so thought it was safer to do the same [i.e on type StateOverrides](https://github.com/pimlicolabs/permissionless.js/blob/2bd149b5f83d8e6d07cfbe57c7ea5b18369fc84b/packages/permissionless/types/bundler.ts#L117)